### PR TITLE
feat(webext/manifest): Add `manifest_version` values

### PR DESCRIPTION
**Pull request by ExE-Boss**
This adds all known `manifest_version` values.

### CODE REVIEW

File: webextensions/manifest/manifest_version.json [hunk 1]
```diff
@@ -18,7 +18,89 @@
...
+            }
+          }
+        },
+        "1": {
+          "__compat": {
+            "description": "Version 1",
+            "support": {
+              "chrome": {
+                "version_added": "4",
+                "version_removed": true
```
Comment (ExE-Boss) [inline webextensions/manifest/manifest_version.json]: needinfo?(@jpmedley): In which version did **Google&nbsp;Chrome** drop support for&nbsp;extension&nbsp;Manifest&nbsp;v1?

File: webextensions/manifest/manifest_version.json [hunk 2]
```diff
@@ -18,7 +18,89 @@
               "version_added": "48"
             },
             "opera": {
-              "version_added": true
+              "version_added": "15"
+            }
+          }
+        },
+        "1": {
```
Comment (vinyldarkscratch) [inline webextensions/manifest/manifest_version.json]:
```suggestion
        "v1": {
```

File: webextensions/manifest/manifest_version.json [hunk 3]
```diff
@@ -18,7 +18,89 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": false,
+              "deprecated": true
+            }
+          }
+        },
+        "2": {
```
Comment (vinyldarkscratch) [inline webextensions/manifest/manifest_version.json]:
```suggestion
        "v2": {
```

File: webextensions/manifest/manifest_version.json [hunk 4]
```diff
@@ -18,7 +18,89 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "3": {
```
Comment (vinyldarkscratch) [inline webextensions/manifest/manifest_version.json]:
```suggestion
        "v3": {
```

File: webextensions/manifest/manifest_version.json (hunk 1 above)
Comment (vinyldarkscratch) [inline webextensions/manifest/manifest_version.json]:
```suggestion
                "version_removed": "21"
```

**Pull request merged**